src/egeoip.app.src

```diff
@@ -3,6 +3,12 @@
   {vsn,"1.1"},
   {modules,[]},
   {registered, []},
-  {env, [{dbfile, city}]},
+  {env, [
+      {dbfile, city}%,
+      % dbpath may be left undefined, so as to not run the self-updater
+      %   but if defined, it must be a dat.gz file
+      %{dbpath, "https://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz"}
+      % or e.g. "https://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
```

Comment:
That is still do be decided, I guess. Wherever we put them (I think we might have to consult MaxMind about unexpected use like mirroring, ...) the option will adapt to that (as long as it's gz). But I wanted an example for app.config and chose the one I could best remember :D

In any case, we're downloading a database every 24 hours. Would that provoke throttling?